src/index/am.rs

```diff
@@ -57,6 +62,7 @@ const AM_HANDLER: pgrx::pg_sys::IndexAmRoutine = {
     // for vector index scans without `ORDER BY` clauses a large number
     // and throw errors if someone really wants such a path.
     am_routine.amoptionalkey = true;
+    am_routine.amstrategies = 2;
```

Comment:
It's only used to check validity of operator class when `CREATE OPERATOR CLASS`. We do not support user-defined operator classes so it's not useful.